shared/chat/conversation/messages/system-new-channel/index.tsx

```diff
@@ -27,6 +28,13 @@ const NewChannel = (props: Props) => {
       >
         {message.text}
       </Kb.Markdown>
+      <Kb.Text
+        onClick={onManageChannels}
+        type="BodySmallSemiboldSecondaryLink"
+        style={{color: Styles.globalColors.blueDark}}
+      >
+        Browse other channels
```

Comment:
i thought it would be better to always show this, we could toggle it only if there are multiple though